dist/angular-patternfly.js

```diff
@@ -1670,6 +1683,7 @@ angular.module('patternfly.charts').directive('pfHeatmap', ["$compile", "$window
        }
 
        $scope.data = {
+         dataAvailable: true,
```

Comment:
@yaacov ok, it just seems odd to pass an extra flag to signal whether there's data or not.

In the long run you may want to think if it make sense to drop it everywhere (unless I missed a specific use-case).